src/createServer.js

```diff
@@ -1,3 +1,43 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+'use strict';
+
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { normalizeURL } = require('./normalizeUrl');
+const { validateErrors } = require('./validateErrors');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const paramsToConvert = normalizeURL(req.url);
+
+    const errors = validateErrors(...paramsToConvert);
+
+    if (errors.length > 0) {
+      res.statusCode = 400;
+      res.statusText = 'Bad request';
+
+      res.end(JSON.stringify({
+        errors,
+      }));
+
+      return;
+    }
+
+    const convertedText = convertToCase(...paramsToConvert);
+
+    res.statusCode = 200;
+    res.statusText = 'OK';
+
+    res.end(JSON.stringify({
+      originalCase: convertedText.originalCase,
+      targetCase: paramsToConvert[1],
+      originalText: paramsToConvert[0],
+      convertedText: convertedText.convertedText,
+    }));
```

Comment:
You can create a function `sendResponse`(res, data, status code), cause you send respond in 2 places and code almost the same

Reply:
And move `res.setHeader('Content-Type', 'application/json')` to that function